.gitlab-ci.yml

```diff
@@ -293,6 +293,9 @@ variables:
 .if_run_e2e: &if_run_e2e
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Do we want to rename this to better mirror the new rule (not in this PR)?
```suggestion
.if_enable_e2e: &if_enable_e2e
```

Reply:
Renamed it `if_auto_e2e` to match the other suggestion